packages/runtime/lib/api.js

```diff
@@ -179,6 +190,40 @@ class RuntimeApi {
     this.#bus.broadcast('runtime:services:stopped')
   }
 
+  async resolveDependencies () {
```

Comment:
Why this method should be public?

Reply:
It is used for tests.

Reply:
prefix with an underscore

Reply:
If you call it when the runtime app starts, there would be no need to call it in tests.  
`loaded.configManager.current.services` - will be already sorted

Reply:
Tests in `packages/runtime/config.test.js` did not start the `RuntimeAPI` so I had to do it explicitly.